data-remote/src/main/java/com/michaldrabik/data_remote/tmdb/model/TmdbImage.kt

```diff
@@ -10,4 +12,15 @@ data class TmdbImage(
   fun isPlain() = iso_639_1 == null
 
   fun isEnglish() = iso_639_1 == "en"
+
+  fun isLanguage(language: String) = iso_639_1 == language
+
+  fun getVoteScore(): Double {
```

Comment:
Just curious about this. Can you shortly tell me like I am 5yo how it picks the "best" voted score? I know there is a Wiki link but its just a bunch of math : )

Reply:
For 5yo xD:
> Think about two boxes of cookie­s 🍪. One is a big box 📦 with many cookies, but not all are tasty 😐. The­ other is a small box 🎁 with few cookies, but e­veryone loves e­ach one ❤️!
> 
> Before, we­ picked the big box 📦 because­ it had more cookies 🍪 . But that was unfair ☹️ to the small box 🎁 with the­ loved cookies 😍.
> 
> Now, we use­ a special formula called Wilson score 🙊. It conside­rs the number of cookies and how much pe­ople enjoy them 😎. So, if the­ small box has cookies everyone­ loves, it might win, even if it has fe­wer cookies 🤓  😲!
>The Wilson score is like a scale ⚖️ that measures not just how many cookies there are, but also how much people like them❣️. It helps us pick the best box of cookies in a fair way☮️, like having a superpower to see which one is the yummiest!😋

On a more serious note xD (and as an addition to the above):

> The formula adjusts the average score based on the number of votes. If there are many votes, the adjustment is small, and we trust the average score more. If there are few votes, the adjustment is larger, and the average score is lowered to reflect the greater uncertainty.
>
> Z-score is a number that represents how many standard deviations a data point is from the mean. For the Wilson score, we usually use a Z-score that corresponds to a 95% confidence level. 
> Depending on this, these adjustments are higher or lower.
> 
> But then we'd have to start talking about statistics xD